tools/sgen/input/FluentUIStyle.yml

```diff
@@ -364,14 +364,14 @@ AP_MSFListCellTokens:
   disclosureInterspace: $Spacing.xxSmall
   disclosureSize: $Icon.size.small
   footnoteFont: $Typography.caption1
-  horizontalCellPadding: $Spacing.medium
+  horizontalCellPadding: $Spacing.xLarge
   iconInterspace: $Spacing.medium
   labelAccessoryInterspace: $Spacing.xSmall
   labelAccessorySize: $Icon.size.xxSmall
   labelColor: $Colors.Foreground.neutral1
   labelFont: $Typography.body
   leadingViewColor: $Colors.Foreground.neutral4
-  leadingViewSize: [ small: $Icon.size.xSmall, medium: $Icon.size.medium, large: $Icon.size.xxLarge, xlarge: 52pt ]
+  leadingViewSize: [ small: $Icon.size.xSmall, medium: $Icon.size.medium, large: $Icon.size.xxLarge ]
```

Comment:
They received feedback that the xLarge was too large & wanted to better align with other list items. It was only used in the PersonaView instance, and because they reduced the size, it was no longer being used anywhere